### PR TITLE
Bump k8s dep to 1.11.3 and mergo to 0.3.6

### PR DESCRIPTION
**Pull request by vdemeester**
Depends on knative/build#410

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -54,6 +54,17 @@ required = [
...
+[[override]]
+  name = "github.com/knative/pkg"
+  # HEAD as of 2018-10-09
+  revision = "ba1a828cd571f605684f9a4506e85bc13cb65b8d"
+
+[[constraint]]
+  name = "github.com/knative/build"
+  # HEAD as of 2018-10-09
+  revision = "0ee769242233fbcff5b36953c55c9cea4f864a9c"
+  source = "github.com/vdemeester/build"
```
Comment (ImJasonH) [inline Gopkg.toml]: Remove this line.

Comment (vdemeester) [reply Gopkg.toml]: @ImJasonH that's why I put `[wip]` in the PR title :angel:

**Comment by bobcatfish** [comment]
Thanks for taking care of this @vdemeester !

Does this have any implication on the requirements of the actual kube cluster we run this on?

**Comment by vdemeester** [comment]
@bobcatfish doesn't change any requirements (it's just a minor version bump)… 
I'm also preparing the 1.12.x bump, that might have some implication (but hopefully none)

**Comment by vdemeester** [comment]
Updated now that knative/build#410 is merged :angel:

**Pull request merged**